.github/workflows/bump-deps.yaml

```diff
@@ -0,0 +1,69 @@
+# Description: Run this by visiting https://github.com/brave/brave-core/actions/workflows/bump-deps.yaml
+# and clicking the `Run workflow` button on the top-right.
+# You can also run it from cli by: `gh workflow run bump-deps.yaml -f ref=commit-hash`
+name: Bump DEPS
+on:
+  workflow_dispatch:
+    inputs:
+      dep:
+        description: "Dependency"
+        required: true
+        type: string
+        default: "vendor/web-discovery-project"
+      ref:
+        description: "New ref for dependency"
+        required: true
+        type: string
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  auto-bump:
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: "0"
+
+      - name: auto-bump
+        run: |
+          set -x
+
+          # git config user.email "${{ github.actor }}@users.noreply.github.com"
+          # git config user.name "${{ github.actor }}"
+          # git config push.default simple
+          gh api /repos/brave/web-discovery-project/commits/main -q .sha
+
+          SED=sed
+          DATE=date
```

Comment:
Let's drop the variables before merging

Reply:
I dropped date bec it's no longer needed.
I prefer to leave SED bec I'll occasionally need to run this locally to debug an issue and on MacOS the provider is gsed. I don't think it hurts us to leave this parameterized for now